content/en/armory-enterprise/quickspin/quickstart-quick-spin.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Quick Spin Quick Start 
+linkTitle: "Start using Quick Spin"
+no_list: true
+weight: 5
+description: >
+  Use this tutorial to lean how to create and manage a simple Spinnaker workflow.
+exclude_search: true
+---
+## Abstract
+This tutorial demonstrates how to:
+- Create a Spinnaker application
+- Create a Jenkins instance and add it to your application
+- Deploy and manage a Spinnaker pipeline for your application
+- Deploy updates to the application
+- Monitor the health of your application
+
+## Prerequisites
+Before you begin make sure you have installed Quick Spin and have an active instance running on `locahost:9000` as described in: TBD.
+
+- You need an active github account
+- Your favorite code editor with some JavaScript support enabled
+- 
+
+## Run the Spinnaker prep pipeline
+
+## Create kub cluster
```

Comment:
```suggestion
## Create a Kubernetes cluster
```
??  Is K8s included with Quick Spin? If not, having access to a K8s cluster should be listed in the `Before you begin` section.